README.md

```diff
@@ -1,48 +1,61 @@
+1. A GitHub project [Secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets#creating-encrypted-secrets) named `CR_TOKEN` with the value of a GitHub [personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line#creating-a-token)
+    * The token must have `repo` scope
+    * The token's user must have write access to the project
+    * To mitigate risk you may wish to limit the token to a single project by creating a [machine user](https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users)
+    * Please note the personal access token is required because of an [Actions bug](https://github.community/t5/GitHub-Actions/Github-action-not-triggering-gh-pages-upon-push/m-p/31266/highlight/true#M743), and will hopefully be unnecessary in the future
+1. Create a workflow `.yml` file in your `.github/workflows` directory. An [example workflow](#example-workflow) is available below. For more information, reference the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file)
 
 ### Inputs
 
-For more information on these inputs, see the [API Documentation](https://developer.github.com/v3/repos/releases/#input)
+For more information on inputs, see the [API Documentation](https://developer.github.com/v3/repos/releases/#input)
 
 - `charts-dir`: The charts directory
-- `charts-repo-url`: The GitHub Pages URL to the charts repo (default: `https://<owner>.github.io/<repo>`)
+- `charts-repo-url`: The GitHub Pages URL to the charts repo (default: `https://<owner>.github.io/<project>`)
 - `token`: The GitHub token
 
 ### Example workflow
 
-Create a workflow (eg: .github/workflows/chart-testing.yml see Creating a Workflow file):
+Create a workflow (eg: `.github/workflows/release.yml`. replace `<owner>` and `<project>` with your GitHub project's owner and project name):
 
 ```yaml
-name: chart actions
+name: Release Charts
 
 on:
   push:
-    branches:    
-      - master 
+    branches:
+      - master
 
 jobs:
-  test:
+  release:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v1
+
+      - name: Configure Git
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email noreply@github.com
+
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1
+        uses: helm/chart-releaser-action@master
+        env:
+          CR_TOKEN: "${{ secrets.CR_TOKEN }}"
         with:
-          token: "${{ secrets.GITHUB_TOKEN }}"
+          owner: <owner>
+          repo: <project>
```

Comment:
Remove both. That's obsolete. The action does that based on the `GITHUB_REPOSITORY` env var.

https://github.com/scottrigby/chart-releaser-action/blob/master/main.sh#L24-L25

Reply:
Removed